ansible/playbooks/vars/hana-vars.yaml

```diff
@@ -0,0 +1,26 @@
+# REQUIRED_PARAMETERS #
```

Comment:
is it fine to have this file included, seems like the one in gitignore but with dash in place of underscore

Reply:
Good spot!  I'll change it :)